src/OrdinaryDiffEq.jl

```diff
@@ -198,7 +198,7 @@ module OrdinaryDiffEq
          Kvaerno3, KenCarp3, Cash4, Hairer4, Hairer42, SSPSDIRK2, Kvaerno4,
          Kvaerno5, KenCarp4, KenCarp5, ESDIRK54I8L2SA
 
-  export MagnusMidpoint, LinearExponential
+  export MagnusMidpoint, LinearExponential, MMUT
```

Comment:
Let's call it MagnusLeapfrog: much more common.

Reply:
@ChrisRackauckas Done !